File: java/aula03/exercicio2-rodizio/src/exercicio2/rodizio/Exercicio2Rodizio.java
package exercicio2.rodizio;

import java.util.Scanner;

public class Exercicio2Rodizio {

    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);
        int numeroPlaca, finalPlaca;
        
        System.out.println("Digite o número da placa:");
        numeroPlaca = entrada.nextInt();
        
        finalPlaca = numeroPlaca % 10;
        
        switch(finalPlaca){
            case 1:
            case 2:
                System.out.println("Segunda-feira");
                break;
            case 3:
            case 4:
                System.out.println("Terça-feira");
                break;
            case 5:
            case 6:
                System.out.println("Quarta-feira");
                break;
            case 7:
            case 8:
                System.out.println("Quinta-feira");
                break;
            case 9:
            case 0:
                System.out.println("Sexta-feira");
                break;
        }
      }
    }
   

File: Python/aula3/exercicio2-rodizio.py
# Solicita o número da placa
numeroPlaca = int(input("Digite o número da placa do veículo: "))
finalPlaca = numeroPlaca % 10

if finalPlaca == 1 or finalPlaca == 2:
    print("Segunda-feira");
elif finalPlaca == 3 or finalPlaca == 4:
    print("Terça-feira");
elif finalPlaca == 5 or finalPlaca == 6:
    print("Quarta-feira");
elif finalPlaca == 7 or finalPlaca == 8:
    print("Quinta-feira");
else:
    print("Sexta-feira");

# importante:
# Em Python, o comando:if finalPlaca == 1 or finalPlaca == 2: pode ser substituído pelo: if finalPlaca in (1,2)!